.github/renovate.json

```diff
@@ -1,19 +1,46 @@
 {
+  "extends": [
+    "config:base"
+  ],
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "enabledManagers": ["custom.regex"],
   "prConcurrentLimit": 50,
-  "prHourlyLimit": 20,
+  "prHourlyLimit": 50,
   "branchConcurrentLimit": 50,
+  "separateMinorPatch": true,
   "labels": ["renovate"],
+  "logLevelRemap": [
+    {
+      "matchMessage": "/^Custom manager fetcher/",
+      "newLogLevel": "info"
+    }
+  ],
   "packageRules": [
     {
+      "matchDatasources": ["docker", "custom.deb1804", "custom.deb2004", "custom.deb2204"],
+      "matchUpdateTypes": [
+        "minor"
+      ],
+      "automerge": false,
+      "enabled": false
+    },
+    {
+      "matchDatasources": ["docker", "custom.deb1804", "custom.deb2004", "custom.deb2204"],
       "matchUpdateTypes": [
-        "minor",
         "patch",
         "pin",
         "digest"
       ],
-      "automerge": false
+      "automerge": false,
+      "enabled": true
+    },
+    {
+      "matchPackageNames": ["moby-runc", "moby-containerd"],
+      "assignees": ["devinwong"]
+    },
+    {
+      "matchPackageNames": ["moby-runc", "moby-containerd"],
```

Comment:
can't this me merged with the previous entry, ? the matcher is the same

Reply:
yes. done.